patches/karma-safari-launcher+1.0.0.patch

```diff
@@ -0,0 +1,152 @@
+diff --git a/node_modules/karma-safari-launcher/index.js b/node_modules/karma-safari-launcher/index.js
```

Comment:
Same note on this as the other PR, would like to have a document in the repo explaining why the patch is needed and what conditions could make it not necessary in the future.

Reply:
is this addressed by the patch README changes?

Reply:
Yes